src/ansi.rs

```diff
@@ -612,6 +612,9 @@ pub trait Handler {
     /// Clear tab stops.
     fn clear_tabs(&mut self, _mode: TabulationClearMode) {}
 
+    /// Reset tab stops to every `interval` column.
+    fn reset_tabs(&mut self, _interval: u16) {}
```

Comment:
I'm not sure `reset` is helpful here, there's no guarantee it ever was 8 to begin with.

I think this function should either be `set_tabs/tabstops`, or something even more specific like setting it to every 8 columns.

Reply:
I've changed the function name to `set_tabs`